marlowe-playground-client/src/Simulation.purs

```diff
@@ -122,9 +124,9 @@ holesPane selectedHole (Holes holes) =
           holesGroup
       ]
   where
-  sortHoles = compare `on` (head <<< snd)
+  sortHoles = compare `on` (head <<< Set.toUnfoldable <<< snd)
```

Comment:
While you're here, it's tempting to refactor this a little. I always find mixed `let` and `where` a bit odd.